stellargraph/data/explorer.py

```diff
@@ -789,3 +793,136 @@ def _check_neighbourhood_sizes(self, in_size, out_size):
             self._raise_error(
                 "The number of hops for the in and out neighbourhoods must be the same."
             )
+
+
+@experimental(reason="requires more thorough testing and documentation")
+class TemporalRandomWalk(GraphWalk):
```

Comment:
This doesn't appear in the documentation. I think you'll need to add it to `docs/api.txt`:

https://github.com/stellargraph/stellargraph/blob/df198b8218d7d6ba06dbd85a12edafb71f6af591/docs/api.txt#L11-L15